notebooks/nightly_report/nightly_report_ts_version.ipynb

```diff
@@ -1,4 +1,4 @@
~{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #332.    
</pre><p>Addding  eg &nbsp;<code>seq_num_corr = event["visitId"]</code> gives us the seqNum from which correction was derived.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>